modeshape-jcr/src/main/java/org/modeshape/jcr/mimetype/tika/TikaNameOnlyDetector.java

```diff
@@ -61,7 +63,8 @@ protected void initDetector( ClassLoader loader ) {
     @Override
     protected String detect( InputStream inputStream, Metadata metadata ) {
         try {
-            // we never care about the stream here
+            // we never care about the stream here, but close it anyway
+            IoUtil.closeQuietly(inputStream);
```

Comment:
I happen to get this kind of S3-related warnings in relation to this unused stream:
```Not all bytes were read from the S3ObjectInputStream, aborting HTTP connection. This is likely an error and may result in sub-optimal behavior. Request only the bytes you need via a ranged GET or drain the input stream after use.```

Comes from https://github.com/aws/aws-sdk-java/blob/master/aws-java-sdk-s3/src/main/java/com/amazonaws/services/s3/internal/S3AbortableInputStream.java#L159.